src/site/content/en/handbook/style/index.md

```diff
@@ -0,0 +1,158 @@
+
+{% Compare 'worse', 'Don’t' %}
+> You can do _x_ by clicking **Run Audit**.
+
+The declarative mood ("you can…") makes it less clear that the reader needs to take an action.
+{% endCompare %}
+
+{% Compare 'better', 'Do' %}
+> Click **Run Audit** to do _x_.
+
+The imperative "click" makes it obvious that an action is needed.
+{% endCompare %}
+
+When talking about technologies or tools, focus on what they let the reader do, not on their features or the work that went into creating them. (Avoiding first-person pronouns helps.)
+
+{% Compare 'worse', 'Don’t' %}
+> We've added _[a cool thing]_ in DevTools.
+
+This sentence emphasizes the work the developers of DevTools did rather than what it lets the reader do.
+{% endCompare %}
+
+{% Compare 'better', 'Do' %}
+> Use _[a cool thing]_ in DevTools to _[do neat stuff]_.
+
+This sentence better focuses on what how readers can meet their goals.
+{% endCompare %}
+
+Address the reader directly rather than imagining a hypothetical third party.
+
+{% Compare 'worse', 'Don’t' %}
+> A seller can choose a variety of shopfront themes.
+
+If the reader isn't a seller, this detail may not seem relevant.
+{% endCompare %}
+
+{% Compare 'better', 'Do' %}
+> You can choose a variety of shopfront themes.
+
+Using second person helps convey that we think this information is relevant to readers. (This strategy assumes that the post has been appropriately targeted to an [audience](/handbook/audience).)
+{% endCompare %}
+
+Provide a way to measure success. How can developers test that they've followed your instruction correctly?
+
+## Keep it simple
+Focus on essential details when explaining a tool or technique. Avoid overwhelming the reader with "real-world" examples that include complexities that aren't required to understand the basic concept. Save those details for supplemental codelabs or follow-up posts. See the "[Code splitting with React.lazy and Suspense](/code-splitting-suspense)" post for an example.
+
+Avoid jargon. If you absolutely _must_ use a term most web.dev readers are unlikely to know, define it in a short aside. See the "[web.dev components](/handbook/web-dev-components#asides)" post for how to include asides. Also check out the "[Inclusion and accessibility](/handbook/inclusion-and-accessibility#use-readable-language)" post for more info about writing readable language.
+
+{% Compare 'worse', 'Don’t' %}
+> In some scripts, graphemes can be visually joined when they're adjacent.
+
+_Script_ and _grapheme_ are very precise, but they're likely to be unfamiliar to most readers and may cause confusion since readers probably more commonly see _script_ referring to a piece of code.
+{% endCompare %}
+
+{% Compare 'better', 'Do' %}
+> In some written languages, certain characters can be visually joined when they're adjacent.
+
+While _written language_ and _character_ are slightly less precise, they should be more familiar and less likely to cause misunderstandings.
+{% endCompare %}
```

Comment:
I wonder if you should add, "Be prepared to push back agains subject matter experts and reviewers." This is excellent advice. Feature owners are often experts in a particular subdomains (web RTC, VR, graphics, to name a few examples). They frequently believe that excluding terms of art from those domains is a **disservice** to readers.